docs/guides/sharing-the-browser-history.md

```diff
@@ -8,7 +8,7 @@ When multiple Feature Apps coexist on the same page, they shouldn't access the
 [browser history API][browser-history-api] directly. Otherwise, they would
 potentially overwrite their respective history and location changes. To enable a
 save access to the history for multiple consumers, the
```

Comment:
Unrelated and not necessarly part of this PR but this should be "To enable safe access to the history for multiple consumers..."